fhir/com.b2international.snowowl.fhir.core/src/com/b2international/snowowl/fhir/core/model/codesystem/Concept.java

```diff
@@ -103,18 +132,54 @@ public Builder definition(final String definition) {
 			return this;
 		}
 		
+		@JsonProperty("designation")
+		@JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
+		public Builder designations(Collection<Designation> designations) {
+			this.designations = designations;
+			return this;
+		}
+		
 		public Builder addDesignation(final Designation designation) {
-			this.designations.add(designation);
+			
+			if (designations == null) {
+				designations = Sets.newHashSet();
```

Comment:
Use either `List` or `SortedSet` to provide consistent serialization of array values.

Reply:
List it is.